packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -415,6 +416,11 @@ const RealAccountSignup = ({
             setIsClosingCreateRealAccountModal(true);
             return;
         }
+        const url = external_url_params?.url;
+        const redirect_action = external_url_params?.redirect_action;
+        if (redirect_action) {
+            window.location.replace(url);
+        }
```

Comment:
this logic is added in two places. Can we make it a fn and call it from different places?

Reply:
ok i will add this method in store and call this ✅